src/main/java/nl/tudelft/jpacman/npc/ghost/moveStrategy.java

```diff
@@ -0,0 +1,7 @@
+package nl.tudelft.jpacman.npc.ghost;
+
+/**
+ * Created by UMONS on 18/04/2016.
+ */
+public interface moveStrategy {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class names should begin with an uppercase character](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2525543484/issues/source?bid=3267846&fileBranchId=3281829#l6)